app/src/main/java/dev/shreyaspatil/covid19notify/utils/ThemeHelper.kt

```diff
@@ -0,0 +1,25 @@
+package dev.shreyaspatil.covid19notify.utils
+
+import android.content.Context
+import android.content.res.Configuration
+import androidx.appcompat.app.AppCompatDelegate
+
+object ThemeHelper {
+    private const val lightMode = "light"
+    private const val darkMode = "dark"
+    private const val batterySaverMode = "battery"
+    const val default = "default"
```

Comment:
Use `Int` constants instead. Also, constants should be always in **UPPERCASE**.
For e.g.
```kt
private const val LIGHT_MODE = 0
```

Reply:
done